README.md

```diff
@@ -4,101 +4,85 @@
 
 - Demonstrate How to Navigate with `bash`
 - Identify our "home directory"
-- Identify my logged-in username with `whoami`
+- Identify your logged-in username with `whoami`
 - Identify the "current working directory" with `pwd` ("print working directory")
-- Navigate up one directory in the file system
-- Change directories using `cd`
+- Navigate directories using `cd`
 
 ## Introduction
 
 Using the CLI (command line interface) might seem like a big challenge to
 first-time users who are afraid of making mistakes that could break their
 computers or ruin their files. Fear not! We'll step you through it.
 
-The command-line interfaces, or "shells", used on OSX, Linux, and the "Windows
-Subsystem for Linux (WSL)," are called `bash`. We'll document how to use the
-`bash` CLI in this module.
+The command-line interfaces, or "shells", used on Linux, older Mac operating
+systems, and the "Windows Subsystem for Linux (WSL)," are called `bash`; newer
+Mac operating systems now come with `zsh` as the default shell. There are other
+shells available as well, but most programmers use shells that are very similar
+to `bash`. The `bash` commands we will cover in this section should prepare you
+to use any of the shells most commonly used by developers, including `zsh`.
```

Comment:
This may be too much detail for beginners, how about:

```markdown
Unix-like operating systems like MacOS, Linux, and Windows Subsystem for Linux
(WSL) all have a command-line interface, or "shell" application. While there are
several different shell applications, like `zsh` (the default for MacOS) and
`bash` (the default for Ubuntu in WSL), all these shell applications share a set
of common commands which developers are familiar with. In this section, we'll
cover some of the most common commands used by developers.
```